### ISSUE TITLE
Highlight current speaker

### ISSUE THREAD

**Issue by bennyboer** [bennyboer/bbb_app#21]
In the web app there are blue bubbles that are highlighted when someone speaks.

**Comment by bennyboer** [comment]
Duplicate of #45 -> closing this one

**Comment by bennyboer** [comment]
Not a duplicate lul 🐧

**Comment by bennyboer** [comment]
Blocked by #82

**Comment by Gleydar** [comment]
Kannst gerne in meinem Branch mal schauen, da sollte das schon drinnen sein ;)
